common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2543,7 +2543,7 @@ StScrollBar {
   }
 }
 .login-dialog-not-listed-label {
-  @include fontsize($font-size * 0.9);
+  @include fontsize($font-size);
```

Comment:
I'm not sure about this. The _Not listed_ is of lesser importance and it makes sense to use noticeably smaller font with it. IMHO this makes it too large compared to the user selection button texts.

Reply:
Ok, font size reverted to smaller text